scripts/ci/docker-compose/base.yml

```diff
@@ -72,3 +72,4 @@ services:
       - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
     ports:
       - "${WEBSERVER_HOST_PORT}:8080"
+      - "5555:5555" # Flower default port
```

Comment:
@potiuk not sure if we need to make it configurable or not

Reply:
It's flexible for user when port's in use.

Reply:
True, same is for webserver port but we are still hardcoding it. I think this is ok for the development environment

Reply:
Yep. Those are default ports, but you can always override them via the variables if for whatever reason you cannot use the default ones. That was the main reason why they are set as variables.